frontend/src/component/context/ContextList/ContextList.tsx

```diff
@@ -24,12 +24,17 @@ import { ContextActionsCell } from './ContextActionsCell/ContextActionsCell';
 import { Adjust } from '@mui/icons-material';
 import { IconCell } from 'component/common/Table/cells/IconCell/IconCell';
 import { Search } from 'component/common/Search/Search';
+import useUiConfig from 'hooks/api/getters/useUiConfig/useUiConfig';
+import { TextCell } from 'component/common/Table/cells/TextCell/TextCell';
+import theme from 'themes/theme';
+import { Box } from '@mui/material';
 
 const ContextList: VFC = () => {
```

Comment:
In a separate PR I'd increase the test coverage for the context list if we don't have any tests in this area. I'd only test the new behavior we're adding which is the usage numbers.